cloudpak/1.0.1/_connectors/kc-sink-elastic/Licensing and support.md

```diff
@@ -0,0 +1,22 @@
+---
+order: 2
```

Comment:
Same comment for all these "kc-sink-" directories.  I think they are from event-streams.